content/docs/how-to-guides/using-branches.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How to Use Branches
+---
+
+Now that you have access to [branching](https://neon.tech/docs/conceptual-guides/branching.md) on Neon, let's look at how you can use it.
+
+Branching allows you to create a copy of your data so that you can modify a version of your project data without changing your production data.
+
+A new branch has all the same data that your project had when you first created the branch, but future changes to the data in your project won't update your branch.
+
+Below is a guide to using branching in your project.
+
+Branches currently appear as a whole new project in the Neon UI and API, but when branches become available to all users, you'll be able to find them within their parent project.
+
+---
+Let's look at how you can create branches using either the Neon Dashboard or the Neon API.
+
+Create a Project on Neon and Add Your Data
+===============
+Log in to Neon and create a project on Neon in the UI: https://console.neon.tech
+Add some data using the SQL Editor in the UI, psql, or any other Postgres client.
+
+Our docs have more information about how to create a project :
+https://neon.tech/docs/cloud/getting-started/
+https://neon.tech/docs/cloud/tutorials/
+https://neon.tech/docs/quickstart/postgres/
+
+Once you have a project with data in it, you can click the `Create branch` button in your project Dashboard to finish creating your branch.
+Alternatively, you can use the Neon API to create your branch.
+
+Create a Branch Using the API
+======================================
+Get familiar with using API keys on Neon: https://neon.tech/docs/get-started-with-neon/using-api-keys/
+The Neon API Reference: https://console.neon.tech/api-docs
+To create a branch, you can use the API method:
+
+	POST /projects/{project_id}/branches
+
+Here's an example of a curl command that you can use to create a branch: 
+
+```
+curl -o - -X POST -H 'Authorization: Bearer ...' https://console.neon.tech/api/v1/clusters/ancient-haze-985396/branches
+```
+
+To use this curl command, change the placeholder project name `ancient-haze-985396` to the name of the project that you want to create a branch from.
+In the response, you will receive new project info info, with the name that looks like
+`$parent_project_id-branch-...`.
+Currently branches appear as a separate project in the Dashboard UI, this will change in the future when branches are publically available.
```

Comment:
```suggestion
Currently, a branch appears as separate Project on the **Dashboard** tab in the Neon Console. This will change in the future when the branching feature becomes generally available.
```